src/notation/internal/notationselectionrange.cpp

```diff
@@ -164,8 +164,6 @@ mu::engraving::Segment* NotationSelectionRange::rangeStartSegment() const
 {
     mu::engraving::Segment* startSegment = score()->selection().startSegment();
 
-    startSegment->measure()->firstEnabled();
```

Comment:
doesn't do anything useful and crashes if startSegment is nullptr which is definitely can be in some cases